### ISSUE TITLE
 Cannot install tts-autolabel

### ISSUE THREAD

**Issue by xueqing0622** [KevinWang676/Bark-Voice-Cloning#70]
Collecting tts-autolabel
  Using cached https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.1.1-py3-none-any.whl (80 kB)
Discarding https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.1.1-py3-none-any.whl (from https://modelscope.oss-cn-beijing.aliyuncs.com/releases/repo.html): Requested tts-autolabel from https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.1.1-py3-none-any.whl has inconsistent version: expected '1.1.1', but metadata has '1.1.2'
  Using cached https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.0.1-py3-none-any.whl (63 kB)
  Using cached https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.0-py3-none-any.whl (62 kB)
ERROR: Cannot install tts-autolabel==1.0, tts-autolabel==1.0.1, tts-autolabel==1.1.2, tts-autolabel==1.1.7 and tts-autolabel==1.1.8 because these package versions have conflicting dependencies.

**Comment by kingki663** [comment]
I also encountered the same problem, have you solved it?

**Comment by xueqing0622** [comment]
no, I even can't find it in https://pypi.org/search/?q=autolabel&o=

**Comment by xueqing0622** [comment]
still not work with create conda env:
My python is 3.10, my stable diffusion use it: maybe I should install python 3.11 and follow your step:

Collecting librosa (from tts-autolabel)
  Using cached https://mirrors.aliyun.com/pypi/packages/e2/a2/4f639c1168d7aada749a896afb4892a831e2041bebdcf636aebfe9e86556/librosa-0.10.1-py3-none-any.whl (253 kB)
Collecting numpy<=1.23.1 (from tts-autolabel)
  Using cached https://mirrors.aliyun.com/pypi/packages/8b/11/75a93826457f94a4c857a38ea3f178915f27ff38ffee1753e36994be7810/numpy-1.23.1-cp310-cp310-win_amd64.whl (14.6 MB)
Collecting sox (from tts-autolabel)
  Using cached https://mirrors.aliyun.com/pypi/packages/3f/67/1810e9a69956eb236967b7174c11fd8d8c2cdab051509286f72e6c7e147e/sox-1.4.1-py2.py3-none-any.whl (39 kB)
Requirement already satisfied: pyyaml in c:\users\admin\appdata\roaming\python\python310\site-packages (from tts-autolabel) (6.0)
Requirement already satisfied: tqdm in c:\programdata\miniconda3\envs\venvbart\lib\site-packages (from tts-autolabel) (4.65.0)
Requirement already satisfied: scipy in c:\programdata\miniconda3\envs\venvbart\lib\site-packages (from tts-autolabel) (1.11.1)
INFO: pip is looking at multiple versions of tts-autolabel to determine which version is compatible with other requirements. This could take a while.
Collecting tts-autolabel
  Using cached https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.1.7-py3-none-any.whl (116 kB)
  Using cached https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.1.2-py3-none-any.whl (80 kB)
Requirement already satisfied: numpy in c:\programdata\miniconda3\envs\venvbart\lib\site-packages (from tts-autolabel) (1.24.3)
Collecting typeguard (from tts-autolabel)
  Downloading https://mirrors.aliyun.com/pypi/packages/18/01/5fc45558268ced46d86292763477996a3cdd505567cd590a688e8cdc386e/typeguard-4.1.5-py3-none-any.whl (34 kB)
Collecting kaldi-native-fbank (from tts-autolabel)
  Using cached https://mirrors.aliyun.com/pypi/packages/fe/d4/c91cfac45bcebf26056bf721ab9303ede62329c0bc192f526591a54992d7/kaldi_native_fbank-1.18.5-cp310-cp310-win_amd64.whl (231 kB)
Collecting pypinyin (from tts-autolabel)
  Downloading https://mirrors.aliyun.com/pypi/packages/00/fc/3e82bf38739a7b2c4f699245ce6c84ff254723c678c2cdc5d2ecbddf9afb/pypinyin-0.49.0-py2.py3-none-any.whl (1.4 MB)
     ---------------------------------------- 1.4/1.4 MB 21.8 MB/s eta 0:00:00
Collecting g2p-en (from tts-autolabel)
  Downloading https://mirrors.aliyun.com/pypi/packages/d7/d9/b77dc634a7a0c0c97716ba97dd0a28cbfa6267c96f359c4f27ed71cbd284/g2p_en-2.1.0-py3-none-any.whl (3.1 MB)
     ---------------------------------------- 3.1/3.1 MB 16.6 MB/s eta 0:00:00
Collecting tts-autolabel
  Using cached https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.1.1-py3-none-any.whl (80 kB)
Discarding https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.1.1-py3-none-any.whl (from https://modelscope.oss-cn-beijing.aliyuncs.com/releases/repo.html): Requested tts-autolabel from https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.1.1-py3-none-any.whl has inconsistent version: expected '1.1.1', but metadata has '1.1.2'
  Using cached https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.0.1-py3-none-any.whl (63 kB)
  Using cached https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.0-py3-none-any.whl (62 kB)
ERROR: Cannot install tts-autolabel==1.0, tts-autolabel==1.0.1, tts-autolabel==1.1.2, tts-autolabel==1.1.7 and tts-autolabel==1.1.8 because these package versions have conflicting dependencies.

The conflict is caused by:
    tts-autolabel 1.1.8 depends on nls-fa
    tts-autolabel 1.1.7 depends on nls-fa
    tts-autolabel 1.1.2 depends on nls-pykaldi
    tts-autolabel 1.0.1 depends on nls-pykaldi
    tts-autolabel 1.0 depends on nls-pykaldi

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts

**Comment by MingjunHu** [comment]
同样的问题，windows系统安装时

**Comment by kingki663** [comment]
I install it in WSL2-Ubuntu22.04, use python3.10

**Comment by sailorsale** [comment]
windows  python3.10  同样出现这个问题
Collecting tts-autolabel
  Using cached https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.1.1-py3-none-any.whl (80 kB)
Discarding https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.1.1-py3-none-any.whl (from https://modelscope.oss-cn-beijing.aliyuncs.com/releases/repo.html): Requested tts-autolabel from https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.1.1-py3-none-any.whl has inconsistent version: expected '1.1.1', but metadata has '1.1.2'
  Using cached https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.0.1-py3-none-any.whl (63 kB)
  Using cached https://modelscope.oss-cn-beijing.aliyuncs.com/releases/dependencies/kantts/tts_autolabel-1.0-py3-none-any.whl (62 kB)
ERROR: Cannot install tts-autolabel==1.0, tts-autolabel==1.0.1, tts-autolabel==1.1.2, tts-autolabel==1.1.7 and tts-autolabel==1.1.8 because these package versions have conflicting dependencies.

The conflict is caused by:
    tts-autolabel 1.1.8 depends on nls-fa
    tts-autolabel 1.1.7 depends on nls-fa
    tts-autolabel 1.1.2 depends on nls-pykaldi
    tts-autolabel 1.0.1 depends on nls-pykaldi
    tts-autolabel 1.0 depends on nls-pykaldi

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
